static/app/components/group/tagFacets/tagFacetsDistributionMeter.tsx

```diff
@@ -152,32 +159,40 @@ function TagFacetsDistributionMeter({
 
   function renderLegend() {
     return (
-      <LegendContainer>
+      <LegendContainer aria-expanded={expanded}>
         {topSegments.map((segment, index) => {
           const pctLabel = Math.floor(percent(segment.count, totalValues));
           const unfocus = !!hoveredValue && hoveredValue.value !== segment.value;
           const focus = hoveredValue?.value === segment.value;
-          return (
-            <Link
-              key={`segment-${segment.name}-${index}`}
-              to={segment.url}
-              aria-label={t(
-                'Add the %s %s segment tag to the search query',
+          const isOtherSegment =
+            index === topSegments.length - 1 && segment.value === 'other';
+          const linkLabel = isOtherSegment
+            ? t(
+                'Other %s tag values, %s of all events. View all tags.',
```

Comment:
"other" label isn't clickable in issue details (looks like the link is an empty string)
also in all cases, clicking on these labels in the issue details page take you to the tag details page so this aria label is misleading

Reply:
👍 Handling this in an upcoming pr!